attalos/imgtxt_algorithms/main.ipynb

```diff
@@ -0,0 +1,168 @@
+{
```

Comment:
@kylemvz , @karllab41 I think we should strongly discourage everyone from using this notebook specifically (in general I really like the notebooks). My concern here is that this notebook is basically the same as the scripted version but without the logging. I worry that it will get used for things like searching the parameter space (which was we would definitely want to capture in Sacred)

That said, I'm fine with it staying in